CONTRIBUTING.md

```diff
@@ -11,18 +11,31 @@ You can build and run virtual users on MacOS, Windows or Linux. You'll need JDK
 
 ## Submitting changes
  
-Pull requests, issues and comments are welcome. For pull requests:
+Pull requests, issues and comments are welcome. Below checklist can help you to go smoothly through the review.
+Please remember that common sense applies here. 
+
+### Pull Request Readiness readiness checklist
+
+  * [ ] Includes tests to exercise the new behavior and bugs.
+  * [ ] Follows the existing style.
+  * [ ] Git commit message is detailed and includes context behind the change.
+  * [ ] If the change is related to an existing Bug Report or Feature Request,
+  the issue number is referenced
+  * [ ] For bigger changes, make sure you start a discussion first by creating 
+  an issue and explaining the intended
+   change.
 
-  - Add tests for new features and bug fixes
-  - Follow the existing style
-  - Separate unrelated changes into multiple pull requests
-  
 See the [existing issues](https://ecosystem.atlassian.net/projects/JPERF/issues/?filter=allissues) for things to start contributing.
 
+## Getting Your Pull Request Merged
+
+All Pull Requests must be approved by at least one member of the @atlassian/jpt team
```

Comment:
This is already explained and enforced by GitHub GUI.

Reply:
I usually read CONTRIBUTING.md before I start to write a code. GUI explains rules at the last phase of the contribution.

Reply:
You can read CODEOWNERS before writing code too. IMHO it's a micro-value, not worth denormalizing